doc/headset.md

```diff
@@ -0,0 +1,430 @@
+* `callInfo: {conversationId: string, contactName?: string}` - The conversationId and (possibly) contactName for the call that is outgoing
+
+Returns: a Promise containing `void`
+
+#### `endCurrentCall()`
+Function that calls the headset library's `endCall(conversationId)` function ONLY IF a conversationId was passed in. If no conversationId was passed in, that means there is no active call so nothing needs to end. This signals the headset device to switch off the answer call button's light to show the active call has ended.
+
+Declaration:
+``` ts
+    endCurrentCall(conversationId: string): Promise<void>;
+```
+
+Params:
+* `conversationId: string` - The conversationId of the call that needs to be ended
+
+Returns: a Promise containing `void`
+
+#### `endAllCalls()`
+Function that calls the headset library's `endAllCalls()` function. This will end not only the current call but all other active calls. This signals the headset device to switch off the answer call buttons's light to show the active calls have all ended
+
+Declaration:
+``` ts
+    endAllCalls(): Promise<void>;
+```
+
+Params: none
+
+Returns: a Promise containing `void`
+
+#### `answerIncomingCalls()`
+Function that calls the headset library's `answerIncomingCall(conversationId)` function. This signals the headset device to switch on the answer call button's light to show the call is now active
+
+Declaration:
+``` ts
+    answerIncomingCall(conversationId: string): Promise<void>;
+```
+
+Params:
+* `conversationId: string` - The conversationId of the call that needs to be answered
+
+Returns: a Promise containing `void`
+
+#### `setMute()`
+Function that calls the headset library's `setMute(isMuted)` function. This signals the headset device to switch on or off (depending on the value of `isMuted`) the mute call button's light to show the call has been muted or unmuted respectively
+
+Declaration:
+``` ts
+    setMute(isMuted: boolean): Promise<void>;
+```
+
+Params:
+* `isMuted: boolean` - value to determine if the device is now muted or unmuted
+
+Returns: a Promise containing `void`
+
+#### `setHold()`
+Function that calls the headset library's `setHold(conversationId, isHeld)` function. This signals the headset device to switch on or off (depending the value of `isHeld`) the hold call button's light to show the call has been held or resumed respectively
+
+Declaration:
+``` ts
+    setHold(conversationId: string, isHeld: boolean): Promise<void>;
+```
+
+Params:
+* `conversationId: string` - The conversationId of the call that needs to be held or resumed
+* `isHeld: boolean` - value to determine if the device is now on hold or resumed
+
+Returns: a Promise containing `void`
+
+--------
+
+### Events
+The SDK Headset Utility does not explicitly emit events itself. It uses RxJS observables (https://rxjs.dev/guide/observable) to emit the events which are then subscribed to within the consuming app. It listens for changes and fires functions that correspond to the events
```

Comment:
NIT: you could create a hyperlink to rxjs rather than showing the whole link. Not necessary, just looks cleaner: 

``` md
[RxJS observables](https://rxjs.dev/guide/observable)
```